### PR TITLE
Changelog: Add the 2.29 release notice

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Just adds a notice to the changelog, but not to the top download popup. 
Hopefully it's enough for a while.

CC @daniel-beck @rtyler

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
@reviewbybees

File: changelog.html [hunk 1]
```diff
@@ -60,7 +60,13 @@
 </ul>
 </div><!--=TRUNK-END=-->
 <h3><a name=v2.29>What's new in 2.29</a> (2016/11/06)</h3>
-<ul class=image>
+<p>
+  <b class="color:red">Warning!</b> This release is not recommended for use due to 
+  <a href="https://issues.jenkins-ci.org/browse/JENKINS-39535">issue 39555</a>
+  and <a href="https://issues.jenkins-ci.org/browse/JENKINS-39535">issue 39414</a>.
```
Comment (batmat) [inline changelog.html]: :-1: twice the same link Oleg

Comment (daniel-beck) [inline changelog.html]: Wrong URL

**Comment by batmat** [comment]
:+1:

File: changelog.html [hunk 2]
```diff
@@ -60,7 +60,13 @@
 </ul>
 </div><!--=TRUNK-END=-->
 <h3><a name=v2.29>What's new in 2.29</a> (2016/11/06)</h3>
-<ul class=image>
+<p>
+  <b class="color:red">Warning!</b> This release is not recommended for use due to 
+  <a href="https://issues.jenkins-ci.org/browse/JENKINS-39535">issue 39555</a>
```
Comment (daniel-beck) [inline changelog.html]: @oleg-nenashev  Still wrong URL ;-)

Comment (oleg-nenashev) [inline changelog.html]: :facepalm:

**Pull request merged**